CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # Change Log
 This file documents all notable changes to juttle-client-library. The release numbering uses [semantic versioning](http://semver.org).
 
-## Unreleased Changes
+## 0.4.0
+
+Released 2016-02-12
+
+### Major Changes
+- change input.getValues to return a promise and text/number input to update onBlur [[#43](https://github.com/juttle/juttle-client-library/pull/43)]
+
+## 0.3.1
+
+Release 2016-02-11
 
 ### Major Changes
-- Add error reporting for invalid JuttleView parameters.
+- Add error reporting for invalid JuttleView parameters. [[#41](https://github.com/juttle/juttle-client-library/pull/41) and [#42](https://github.com/juttle/juttle-client-library/pull/42)]
```

Comment:
I'm worried that I might have to revert this change, perhaps not touch this and I'll this on another commit.